src/main/java/org/terasology/gookeeper/system/HungerSystem.java

```diff
@@ -0,0 +1,203 @@
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.entitySystem.systems.UpdateSubscriberSystem;
+import org.terasology.gookeeper.Constants;
+import org.terasology.gookeeper.component.GooeyComponent;
+import org.terasology.gookeeper.component.HungerComponent;
+import org.terasology.gookeeper.event.GooeyFedEvent;
+import org.terasology.logic.characters.CharacterHeldItemComponent;
+import org.terasology.logic.common.ActivateEvent;
+import org.terasology.logic.common.DisplayNameComponent;
+import org.terasology.logic.delay.DelayManager;
+import org.terasology.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.logic.health.HealthComponent;
+import org.terasology.logic.inventory.InventoryManager;
+import org.terasology.logic.players.LocalPlayer;
+import org.terasology.physics.Physics;
+import org.terasology.registry.In;
+import org.terasology.registry.Share;
+import org.terasology.rendering.nui.NUIManager;
+import org.terasology.rendering.nui.layers.ingame.inventory.GetItemTooltip;
+import org.terasology.rendering.nui.widgets.TooltipLine;
+import org.terasology.utilities.Assets;
+import org.terasology.utilities.random.FastRandom;
+import org.terasology.utilities.random.Random;
+import org.terasology.world.BlockEntityRegistry;
+import org.terasology.world.WorldProvider;
+import org.terasology.worldlyTooltipAPI.events.GetTooltipIconEvent;
+import org.terasology.worldlyTooltipAPI.events.GetTooltipNameEvent;
+
+@RegisterSystem(RegisterMode.AUTHORITY)
+@Share(value = HungerSystem.class)
+public class HungerSystem extends BaseComponentSystem implements UpdateSubscriberSystem {
+
+    @In
+    private WorldProvider worldProvider;
+
+    @In
+    private EntityManager entityManager;
+
+    @In
+    private BlockEntityRegistry blockEntityRegistry;
+
+    @In
+    private LocalPlayer localPlayer;
+
+    @In
+    private InventoryManager inventoryManager;
+
+    @In
+    private Physics physics;
+
+    @In
+    private AssetManager assetManager;
+
+    @In
+    private PrefabManager prefabManager;
+
+    @In
+    private DelayManager delayManager;
+
+    @In
+    private NUIManager nuiManager;
+
+    private static final Logger logger = LoggerFactory.getLogger(HungerSystem.class);
+    private Random random = new FastRandom();
+
+    @Override
+    public void initialise() {
+    }
+
+    @Override
+    public void shutdown() {
+    }
+
+    @Override
+    public void update(float delta) {
+
+    }
+
+    /**
+     * Adds health degradation action for the already captured gooey entities.
+     *
+     * @param event
+     * @param entity
+     * @param gooeyComponent
+     * @param hungerComponent
+     */
+    public void onGooeyActivated(OnAddedComponent event, EntityRef entity, GooeyComponent gooeyComponent, HungerComponent hungerComponent) {
```

Comment:
Is this method supposed to be here? 
It seems to be an event handler without a `@ReceiveEvent`